enterprise_access/apps/api/v1/tests/test_assignment_configuration_views.py

```diff
@@ -374,46 +365,27 @@ def test_create(self):
         """
         Test that create view happy path.  A net-new AsssignmentConfiguration should be created.
         """
+        yet_another_enterprise_uuid = str(uuid4())
         # Set the JWT-based auth to an operator.
         self.set_jwt_cookie([
-            {'system_wide_role': SYSTEM_ENTERPRISE_OPERATOR_ROLE, 'context': str(TEST_ENTERPRISE_UUID)}
+            {'system_wide_role': SYSTEM_ENTERPRISE_OPERATOR_ROLE, 'context': yet_another_enterprise_uuid}
         ])
 
         # Send a create request which should create a net-new AssignmentConfiguration.
-        # Also check that it is linked to the appropriate SubsidyAccessPolicy.
+        # It's possible to create these without linking directly to a policy record.
         list_url = reverse('api:v1:assignment-configurations-list')
-        post_payload = {'subsidy_access_policy': str(self.assigned_learner_credit_policy_standalone.uuid)}
+        post_payload = {'enterprise_customer_uuid': yet_another_enterprise_uuid}
+
         response = self.client.post(list_url, post_payload)
-        assert response.status_code == status.HTTP_201_CREATED
-        self.assigned_learner_credit_policy_standalone.refresh_from_db()
-        assert response.json() == {
-            'uuid': str(self.assigned_learner_credit_policy_standalone.assignment_configuration.uuid),
-            'active': True,
-            'enterprise_customer_uuid': str(TEST_ENTERPRISE_UUID),
-            'subsidy_access_policy': str(self.assigned_learner_credit_policy_standalone.uuid),
-        }
 
-    def test_create_idempotent(self):
```

Comment:
No concerns over dropping the idempotency test?

Reply:
Nope, because we dropped the idempotency!